Project.toml

```
@@ -11,7 +11,7 @@ Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 MultiBroadcastFusionCUDAExt = ["CUDA", "Adapt"]
 
 [compat]
-julia = "^1.10"
+julia = "^1.9"
```

```suggestion
julia = "1.8"
```
I think we can widen to julia 1.8

---

We need 1.9 because the package is using extensions, which were introduced in 1.9

---

Ah, right. We could add support to 1.8 with Requires, but I'm not sure it's worth it.